pytest_hot_reloading/plugin.py

```diff
@@ -138,11 +140,14 @@ def apply_assertion_rewrite(self, ast_func, glb):
             return ast_func
 
         def stash(self, lineno=1, col_offset=0):
+            # monkeypatch: There's an off-by-one bug coming from somewhere in jurigged.
+            #              This affects replaced functions. When line numbers are wrong
+            #              the debugger and inspection logic doesn't work as expected.
             if not isinstance(self.parent, OrigFunctionDefinition):
                 co = self.get_object()
                 if co and (delta := lineno - co.co_firstlineno):
                     delta -= 1  # fix off-by-one
-                    if delta > 0:
+                    if delta != 0:
                         self.recode(jurigged_utils.shift_lineno(co, delta), use_cache=False)
```

Comment:
While debugging I noticed this case and found that delta could be negative. It looked like the negative value, with the off-by-one fix, was correct, so that is why this changed to `!=`